tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -129,6 +129,24 @@ CLRTestExitCode=$?
       <Output TaskParameter="ParamList" PropertyName="_CLRTestParamList"/>
     </GenerateParamList>
 
+<<<<<<< HEAD
```

Comment:
merge confict have not resolved , you need to do it manually, remove the unnecessary lines